samples/bluetooth/iso_broadcast/src/main.c

```diff
@@ -170,7 +170,7 @@ void main(void)
 		iso_send_count++;
 		seq_num++;
 
-		if ((iso_send_count % 100) == 0) {
+		if ((iso_send_count % CONFIG_ISO_PRINT_INTERVAL) == 0) {
```

Comment:
The compare should be 
```suggestion
		if ((iso_send_count % CONFIG_ISO_PRINT_INTERVAL) == 1) {
```

Reply:
I see that the print is on the wrong side of the increment. Fixing this by moving the printout.